pyramid_oereb/lib/records/view_service.py

```diff
@@ -298,3 +284,27 @@ def unique_update_legends(self, legend):
                 break
         if not already_exist:
             self.legends.append(legend)
+
+    def calculate_ns(self):
+        srid = Config.get_crs()
+        if srid == u'epsg:2056':
+            self.min_NS95, self.max_NS95 = self.get_bbox_from_url(self.reference_wms)
+        if srid == u'epsg:21781':
+            self.min_NS03, self.max_NS03 = self.get_bbox_from_url(self.reference_wms)
+
+    @staticmethod
+    def get_bbox_from_url(wms_url):
+        """
+        Parses wms url for BBOX parameter an returns these points as suitable values for ViewServiceRecord.
+        Args:
+            wms_url (str): wms url which includes a BBOX parameter to parse.
+
+        Returns:
+            set of two shapely.geometry.point.Point: min and max coordinates of bounding box.
+        """
+        url, params = parse_url(wms_url)
```

Comment:
Codacy complains about the unused variable 'url'.

What would be the nice way to solve this?

`__, params = parse_url(wms_url)`

or

`params = parse_url(wms_url)[1]`